composer.json

```diff
@@ -10,11 +10,11 @@
         }
     },
     "require": {
-        "php": ">=7.0",
+        "php": "^7.0 || ^8.1",
```

Comment:
Are you intentionally preventing PHP 8.0?

Reply:
That doesn't make much sense. I think I did that because Symfony's PHP requirement jumped from 7 to 8.1.
I changed it to ^8.0.